src/composables/use-event-listener-outside.ts

```diff
@@ -1,43 +1,26 @@
-import { watch, ref } from '@nuxtjs/composition-api'
+import { Ref, ref, watch } from '@nuxtjs/composition-api'
 
-import { contains, getDocument } from '~/utils/reakit-utils/dom'
+import { contains, getDocument, isInDocument } from '~/utils/reakit-utils/dom'
 
-/**
- * @param {Element} target
- */
-function isInDocument(target) {
-  const document = getDocument(target)
-  if (target.tagName === 'HTML') return true
-  return contains(document.body, target)
+type Props = {
```

Comment:
Could use some documentation about this type and it's fields. Also why not make this an `interface`?

```suggestion
interface Props {
```